### PR TITLE
adding unittest framework and some initial tests

### PR DESCRIPTION
**Pull request by brianedmonds90**

### CODE REVIEW

File: test/test_utility_svc.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import unittest
+from app.service.utility_svc import UtilityService
```
Comment (wbooth) [inline test/test_utility_svc.py]: pep008

File: test/test_utility_svc.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+class TestUtilityService(unittest.TestCase):
+
+    # Tests that the encode string function correctly encodes
```
Comment (wbooth) [inline test/test_utility_svc.py]: no comments needed

File: test/test_utility_svc.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        self.assertEqual(UtilityService.encode_string("this is a super string!"),"dGhpcyBpcyBhIHN1cGVyIHN0cmluZyE=")
+
+    # Tests that the decode string function correctly decodes
+    def test_decode_string(self):
+        self.assertEqual(UtilityService.decode_bytes("Ynl0ZXMgdGhhdCBJIG5lZWQgZGVjb2RlZCEgSGVyZSBhcmUgdGhlIGJ5dGVzLiA="),
+                                                     "bytes that I need decoded! Here are the bytes. ")
+
+    # Tests if the encode and decode string functions can encode and decode a string to the original value
+    def test_decode_encode_string_functional(self):
+        mystring = "Speedily say has suitable disposal add boy. On forth doubt miles of child. Exercise joy man " \
```
Comment (wbooth) [inline test/test_utility_svc.py]: keep to 1 line for mystring

File: tests/test_utility_svc.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import unittest
+
+from app.service.utility_svc import UtilityService
+
+
+class TestUtilityService(unittest.TestCase):
+    def test_encode_string(self):
+        self.assertEqual(UtilityService.encode_string("this is a super string!"),"dGhpcyBpcyBhIHN1cGVyIHN0cmluZyE=")
```
Comment (privateducky) [inline tests/test_utility_svc.py]: change double-quotes to single.

Comment (wbooth) [reply tests/test_utility_svc.py]: updated

**Pull request merged**